cmd/kops/create_cluster.go

```diff
@@ -400,17 +388,19 @@ func RunCreateCluster(ctx context.Context, f *util.Factory, out io.Writer, c *Cr
 		return fmt.Errorf("--name is required")
 	}
 
-	cluster, err := clientset.GetCluster(ctx, c.ClusterName)
-	if err != nil {
-		if apierrors.IsNotFound(err) {
-			cluster = nil
-		} else {
-			return err
+	{
+		cluster, err := clientset.GetCluster(ctx, c.ClusterName)
+		if err != nil {
+			if apierrors.IsNotFound(err) {
+				cluster = nil
+			} else {
+				return err
+			}
 		}
-	}
 
-	if cluster != nil {
-		return fmt.Errorf("cluster %q already exists; use 'kops update cluster' to apply changes", c.ClusterName)
+		if cluster != nil {
+			return fmt.Errorf("cluster %q already exists; use 'kops update cluster' to apply changes", c.ClusterName)
+		}
```

Comment:
Any special reason why this gets its own code block?

Reply:
To take `cluster` out of scope from subsequent code.

Reply:
👍